exercises/launchintro-execpols_solution.cpp

```diff
@@ -227,8 +227,8 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 
 // _raja_tensorinit_cuda_start
   using cuda_teams_z_3 = RAJA::expt::LoopPolicy<RAJA::cuda_block_z_direct>;
-  using cuda_global_thread_y_3 = RAJA::expt::LoopPolicy<RAJA::cuda_global_thread_y>;
-  using cuda_global_thread_x_3 = RAJA::expt::LoopPolicy<RAJA::cuda_global_thread_x>;
+  using cuda_global_thread_y_3 = RAJA::expt::LoopPolicy<RAJA::expt::cuda_global_thread_y>;
+  using cuda_global_thread_x_3 = RAJA::expt::LoopPolicy<RAJA::expt::cuda_global_thread_x>;
```

Comment:
if you want you can just revert the changes in `exercises/launchintro-execpols.cpp` and `exercises/launchintro-execpols_solution.cpp` as a different PR will address these changes.

Reply:
I will once that gets merged, otherwise everything fails in CI.